### PR TITLE
Set out-of-tree-vsphere KEP to implemented

### PR DESCRIPTION
**Pull request by frapposelli**
Fixes: #670

/assign @andrewsykim

### CODE REVIEW

File: keps/sig-cloud-provider/20190125-out-of-tree-vsphere.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ approvers:
 editor: TBD
 creation-date: 2019-01-25
 last-updated: 2019-01-25
```
Comment (palnabarun) [inline keps/sig-cloud-provider/20190125-out-of-tree-vsphere.md:19]: Can you also please update the `last-updated` date?

**Comment by frapposelli** [comment]
superseded by #1677

**Pull request closed without merging**